eng/Directory.Packages.props

```
@@ -39,10 +39,10 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(DotNetBuildSourceOnly)' != 'true' and '$(TargetFramework)' != 'net472'">
-    <PackageVersion Include="Microsoft.Build" Version="17.7.2" />
-    <PackageVersion Include="Microsoft.Build.Framework" Version="17.7.2" />
-    <PackageVersion Include="Microsoft.Build.Tasks.Core" Version="17.7.2" />
-    <PackageVersion Include="Microsoft.Build.Utilities.Core" Version="17.7.2" />
+    <PackageVersion Include="Microsoft.Build" Version="17.13.9" />
+    <PackageVersion Include="Microsoft.Build.Framework" Version="17.13.9" />
+    <PackageVersion Include="Microsoft.Build.Tasks.Core" Version="17.13.9" />
+    <PackageVersion Include="Microsoft.Build.Utilities.Core" Version="17.13.9" />
```

This is a problem because 17.13.9 only ships net9.0 and net472. We still use net8.0 most places and the last published build with net8.0 is 17.11.4. 

```suggestion
    <PackageVersion Include="Microsoft.Build" Version="17.11.4" />
    <PackageVersion Include="Microsoft.Build.Framework" Version="17.11.4" />
    <PackageVersion Include="Microsoft.Build.Tasks.Core" Version="17.11.4" />
    <PackageVersion Include="Microsoft.Build.Utilities.Core" Version="17.11.4" />
```